### ISSUE TITLE
Bug Report: 2162-0002(0x4a2)

### ISSUE THREAD

**Issue by 1149271370** [Atmosphere-NX/Atmosphere#2434]
## Bug Report
2162-0002(0x4a2)(which is different from usual) 

### What's the issue you encountered?

After disabling Airplane Mode or attempting to connect to Wi-Fi,The system crashes and prompts me to restart, regardless of whether it's the official system or an emulated one

### How can the issue be reproduced?

The issue can be reproduced simply by entering either the official system or the emulated system, then disabling Airplane Mode or attempting to connect to Wi-Fi.

### Crash Report

[01730689576_010000000000000e.log](https://github.com/user-attachments/files/17613463/01730689576_010000000000000e.log)
[01730689576_010000000000002f.log](https://github.com/user-attachments/files/17613466/01730689576_010000000000002f.log)

### System Firmware Version

19.0.0

### Environment?

- What bootloader (fusèe, hekate, etc) was Atmosphère launched by: hekate 6.2.2
- Official release or unofficial build: 
  - [  unofficial build ]
  - [ If using an unofficial build, include details on where/how you acquired the build. ] 
     - [ Ex: Self-compilation by other]
- Do you have additional kips or sysmodules you're loading: I'm not sure how to verify this option
- Homebrew software installed: [ DBI,DayBreak ]
- EmuMMC or SysNAND:EmuMMC include file-based(maybe?)

### Additional context?

I apologize; this is my first time filling out a troubleshooting form, so my description may not be very accurate. However, I’m eager to resolve this issue and am trying to figure out where the problem lies.

**Comment by impeeza** [comment]
Enable dns MiTM or remove prodinfo blanking

**Comment by 1149271370** [comment]
> Enable dns MiTM or remove prodinfo blanking

![image](https://github.com/user-attachments/assets/2e240e3e-d6df-4373-91b1-2f27148a18a0)
dns MITM seems like it might already be enabled
I will try deleting the files now.

**Comment by 1149271370** [comment]
> > Enable dns MiTM or remove prodinfo blanking
> 
> ![image](https://private-user-images.githubusercontent.com/83260157/382637684-2e240e3e-d6df-4373-91b1-2f27148a18a0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2OTQwMzEsIm5iZiI6MTczMDY5MzczMSwicGF0aCI6Ii84MzI2MDE1Ny8zODI2Mzc2ODQtMmUyNDBlM2UtZDZkZi00MzczLTkxYjEtMmYyNzE0OGExOGEwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA0VDA0MTUzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQyZDcwYTZmMmMwY2IwYWYzNTMzNjFkYzk3MGMzYTg3ZjQ0NWQzYTdhYzRlMjhlMjhkZDljM2RlYmYzMjBjOWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.zHvMo1VK2295RRB5SF6puPwaSYiaT1yWl93qr9MpZ1I) dns MITM seems like it might already be enabled I will try deleting the files now.

![image](https://github.com/user-attachments/assets/6d6b1c56-f705-4502-acfc-1939e43d12bc)

Is it this file in this directory? I deleted it, but the same error still occurs.But I still really appreciate you answering my questions.

**Comment by HeikesFootSlave** [comment]
> > > Enable dns MiTM or remove prodinfo blanking
> > 
> > 
> > ![image](https://private-user-images.githubusercontent.com/83260157/382637684-2e240e3e-d6df-4373-91b1-2f27148a18a0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2OTQwMzEsIm5iZiI6MTczMDY5MzczMSwicGF0aCI6Ii84MzI2MDE1Ny8zODI2Mzc2ODQtMmUyNDBlM2UtZDZkZi00MzczLTkxYjEtMmYyNzE0OGExOGEwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA0VDA0MTUzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQyZDcwYTZmMmMwY2IwYWYzNTMzNjFkYzk3MGMzYTg3ZjQ0NWQzYTdhYzRlMjhlMjhkZDljM2RlYmYzMjBjOWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.zHvMo1VK2295RRB5SF6puPwaSYiaT1yWl93qr9MpZ1I) dns MITM seems like it might already be enabled I will try deleting the files now.
> 
> ![image](https://private-user-images.githubusercontent.com/83260157/382638640-6d6b1c56-f705-4502-acfc-1939e43d12bc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA3MDcwNTUsIm5iZiI6MTczMDcwNjc1NSwicGF0aCI6Ii84MzI2MDE1Ny8zODI2Mzg2NDAtNmQ2YjFjNTYtZjcwNS00NTAyLWFjZmMtMTkzOWU0M2QxMmJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA0VDA3NTIzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRiZDIxM2FmZjQxNmFjYWRiNzgyMjI0MWM2YjAzMWU2MTc2OWZjMmZhNDhjMGM3MmFiMWY0Y2RmYjBlODEyZGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0._dGRnfBQN0aG7F7bflZMV__KqOCxuUjYudf43KFBTgU)
> 
> Is it this file in this directory? I deleted it, but the same error still occurs.But I still really appreciate you answering my questions.

You deleted the file? Standing Ovations! WHY?
Hope it creates it again at startup because it is in a BACKUP Folder!!! BACKUP!

@impeeza NEVER said delete those files he said enable dns_mitm in system_settings or remove prodinfo_blanking = exosphere.ini in the root of the SD (if not get it from ams folder in config_templates)

**Comment by 1149271370** [comment]
> > > > Enable dns MiTM or remove prodinfo blanking
> > > 
> > > 
> > > ![image](https://private-user-images.githubusercontent.com/83260157/382637684-2e240e3e-d6df-4373-91b1-2f27148a18a0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2OTQwMzEsIm5iZiI6MTczMDY5MzczMSwicGF0aCI6Ii84MzI2MDE1Ny8zODI2Mzc2ODQtMmUyNDBlM2UtZDZkZi00MzczLTkxYjEtMmYyNzE0OGExOGEwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA0VDA0MTUzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQyZDcwYTZmMmMwY2IwYWYzNTMzNjFkYzk3MGMzYTg3ZjQ0NWQzYTdhYzRlMjhlMjhkZDljM2RlYmYzMjBjOWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.zHvMo1VK2295RRB5SF6puPwaSYiaT1yWl93qr9MpZ1I) dns MITM seems like it might already be enabled I will try deleting the files now.
> > 
> > 
> > ![image](https://private-user-images.githubusercontent.com/83260157/382638640-6d6b1c56-f705-4502-acfc-1939e43d12bc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA3MDcwNTUsIm5iZiI6MTczMDcwNjc1NSwicGF0aCI6Ii84MzI2MDE1Ny8zODI2Mzg2NDAtNmQ2YjFjNTYtZjcwNS00NTAyLWFjZmMtMTkzOWU0M2QxMmJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA0VDA3NTIzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRiZDIxM2FmZjQxNmFjYWRiNzgyMjI0MWM2YjAzMWU2MTc2OWZjMmZhNDhjMGM3MmFiMWY0Y2RmYjBlODEyZGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0._dGRnfBQN0aG7F7bflZMV__KqOCxuUjYudf43KFBTgU)
> > Is it this file in this directory? I deleted it, but the same error still occurs.But I still really appreciate you answering my questions.
> 
> You deleted the file? Standing Ovations! WHY? Hope it creates it again at startup because it is in a BACKUP Folder!!! BACKUP!
> 
> @impeeza NEVER said delete those files he said enable dns_mitm in system_settings or remove prodinfo_blanking = exosphere.ini in the root of the SD (if not get it from ams folder in config_templates)

I'm very sorry; I misinterpreted what @impeeza meant 
Don’t worry, I have a backup of this backup file
Actually, while asking these questions, I handed my Switch over to a technician who can handle this kind of issue. I also shared the above suggestions with the technician, hoping they might help with the repair.
Thanks again to both of you. @impeeza @HeikesFootSlave

**Comment by EGFNF** [comment]
> > > > Enable dns MiTM or remove prodinfo blanking
> > > 
> > > 
> > > ![image](https://private-user-images.githubusercontent.com/83260157/382637684-2e240e3e-d6df-4373-91b1-2f27148a18a0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2OTQwMzEsIm5iZiI6MTczMDY5MzczMSwicGF0aCI6Ii84MzI2MDE1Ny8zODI2Mzc2ODQtMmUyNDBlM2UtZDZkZi00MzczLTkxYjEtMmYyNzE0OGExOGEwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA0VDA0MTUzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQyZDcwYTZmMmMwY2IwYWYzNTMzNjFkYzk3MGMzYTg3ZjQ0NWQzYTdhYzRlMjhlMjhkZDljM2RlYmYzMjBjOWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.zHvMo1VK2295RRB5SF6puPwaSYiaT1yWl93qr9MpZ1I) dns MITM seems like it might already be enabled I will try deleting the files now.
> > 
> > 
> > ![image](https://private-user-images.githubusercontent.com/83260157/382638640-6d6b1c56-f705-4502-acfc-1939e43d12bc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA3MDcwNTUsIm5iZiI6MTczMDcwNjc1NSwicGF0aCI6Ii84MzI2MDE1Ny8zODI2Mzg2NDAtNmQ2YjFjNTYtZjcwNS00NTAyLWFjZmMtMTkzOWU0M2QxMmJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA0VDA3NTIzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRiZDIxM2FmZjQxNmFjYWRiNzgyMjI0MWM2YjAzMWU2MTc2OWZjMmZhNDhjMGM3MmFiMWY0Y2RmYjBlODEyZGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0._dGRnfBQN0aG7F7bflZMV__KqOCxuUjYudf43KFBTgU)
> > Is it this file in this directory? I deleted it, but the same error still occurs.But I still really appreciate you answering my questions.
> 
> You deleted the file? Standing Ovations! WHY? Hope it creates it again at startup because it is in a BACKUP Folder!!! BACKUP!
> 
> @impeeza NEVER said delete those files he said enable dns_mitm in system_settings or remove prodinfo_blanking = exosphere.ini in the root of the SD (if not get it from ams folder in config_templates)

A standing OVATION EVERYONE FOR THE BIGGEST JERK TECH SUPPORT GUY!

If you can't stand the heat STAY OUT OF THE KITCHEN my dude! Stop yelling at people and quit your day job if you can't handle lower intelligence people. Not everyone can be a "genius" like you!

**Comment by HeikesFootSlave** [comment]
> > > > > Enable dns MiTM or remove prodinfo blanking
> > > > 
> > > > 
> > > > ![image](https://private-user-images.githubusercontent.com/83260157/382637684-2e240e3e-d6df-4373-91b1-2f27148a18a0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2OTQwMzEsIm5iZiI6MTczMDY5MzczMSwicGF0aCI6Ii84MzI2MDE1Ny8zODI2Mzc2ODQtMmUyNDBlM2UtZDZkZi00MzczLTkxYjEtMmYyNzE0OGExOGEwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA0VDA0MTUzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQyZDcwYTZmMmMwY2IwYWYzNTMzNjFkYzk3MGMzYTg3ZjQ0NWQzYTdhYzRlMjhlMjhkZDljM2RlYmYzMjBjOWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.zHvMo1VK2295RRB5SF6puPwaSYiaT1yWl93qr9MpZ1I) dns MITM seems like it might already be enabled I will try deleting the files now.
> > > 
> > > 
> > > ![image](https://private-user-images.githubusercontent.com/83260157/382638640-6d6b1c56-f705-4502-acfc-1939e43d12bc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA3MDcwNTUsIm5iZiI6MTczMDcwNjc1NSwicGF0aCI6Ii84MzI2MDE1Ny8zODI2Mzg2NDAtNmQ2YjFjNTYtZjcwNS00NTAyLWFjZmMtMTkzOWU0M2QxMmJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA0VDA3NTIzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRiZDIxM2FmZjQxNmFjYWRiNzgyMjI0MWM2YjAzMWU2MTc2OWZjMmZhNDhjMGM3MmFiMWY0Y2RmYjBlODEyZGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0._dGRnfBQN0aG7F7bflZMV__KqOCxuUjYudf43KFBTgU)
> > > Is it this file in this directory? I deleted it, but the same error still occurs.But I still really appreciate you answering my questions.
> > 
> > 
> > You deleted the file? Standing Ovations! WHY? Hope it creates it again at startup because it is in a BACKUP Folder!!! BACKUP!
> > @impeeza NEVER said delete those files he said enable dns_mitm in system_settings or remove prodinfo_blanking = exosphere.ini in the root of the SD (if not get it from ams folder in config_templates)
> 
> I'm very sorry; I misinterpreted what @impeeza meant Don’t worry, I have a backup of this backup file Actually, while asking these questions, I handed my Switch over to a technician who can handle this kind of issue. I also shared the above suggestions with the technician, hoping they might help with the repair. Thanks again to both of you. @impeeza @HeikesFootSlave

I have to apologies that was a bit harsh!
PRODINFO Data is very sensible and if you loose that and break it you are glad to have a backup!

**Comment by 1149271370** [comment]
Status update: The technician tried replacing some hardware, but it was to no avail, ultimately confirming that the issue lies at the software level. I also tried using some "all-in-one" packages supposedly supporting 19.0.1, but they didn’t work either. I’ve already seen similar cases appearing in some online groups, and this issue seems to be spreading. The technician told me that initializing the official system can resolve the issue, but since it’s impossible to connect to the internet, account data in the official system cannot be transferred to another device. Restoring data from the official (emulated) system to the official system carries a risk of getting banned. For now, I can only hope that Atmosphere will release an update soon and hope it might resolve this issue.

**Comment by SciresM** [comment]
I don't know what's happening in this issue, but it's not an atmosphere problem. I would just fully wipe your nand following an unbricking guide.

**Comment by 1149271370** [comment]
The issue has been resolved. The solution was: after finding firmware compatible with 19.0.1, I updated the emulated system first, then updated the official custom firmware system, which allowed me to connect to the internet. The problem may have been caused by my own oversight, as my official custom firmware system had been previously blocked. Thank you all for your responses and sorry for the trouble—thanks again!
